backend/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module'
+  },
+  parser: 'babel-eslint',
+  env: {
+    node: true,
+    es6: true
+  },
+  plugins: ['prettier'],
+  extends: ['eslint:recommended', 'prettier'],
+  rules: {
+    'no-console': 'off',
+    'prettier/prettier': 'error'
+  }
```

Comment:
Since there is no root eslint config anymore, i'd add `root: true` here (and in the frontend eslintrc).